### ISSUE TITLE
>some results

### ISSUE THREAD

**Issue by Saralyliu** [Jia-Research-Lab/PFENet#37]
> @Saralyliu
> Hi,
> 
> Thanks for your attention.
> 
> The pre-trained weights of resnet-v2 are obtained from the official repo of PSPNet (https://github.com/hszhao/semseg). The difference between the original resnet only lies in the layer0 where the v2 version applies the deep-stem strategy. We used resnet-v2 to reproduce CANet and we got rather comparable results to the ones reported in the paper of CANet.
> 
> The mask used in "supp_feat_4 = self.layer4(supp_feat_3*mask)" is used for screening out the redundant background region, and I remember that it will not affect the performance much, you can try it out by sending feat_3 to layer-4 without the masking operation.
> 
> The another mask used in "tmp_supp_feat_4 = tmp_supp_feat * tmp_mask" is more important, since it is used for the prior calculation.

Thank you for your reply. If I understand correctly, resnet-v2 or resnet-50 is the same for feature extractor? Recently, we run voc group0 with your code, train is the numbers of 5955,val is 1449, and the best mIoU we test is 58.57 at 124 epoch without any modifications. we can't get your 61.7 mIoU in 1-shot case. waiting for your suggestion, thank you!

**Comment by tianzhuotao** [comment]
Please use 1 GPU to train our model.

You can find more details for reproduction at this issue: 

https://github.com/Jia-Research-Lab/PFENet/issues/6

**Comment by Saralyliu** [comment]
> Please use 1 GPU to train our model. The multi-gpu training may lead to performance deduction.
> 
> You can find more details for reproduction in this issue: #6

yes, we use only 1-GPU

**Comment by Saralyliu** [comment]
![QNU$QQB6SN H2_A~VD0(H65](https://user-images.githubusercontent.com/52000910/114326540-e4f30280-9b67-11eb-8256-5c72c66ee965.png)

**Comment by Saralyliu** [comment]
> Please use 1 GPU to train our model. The multi-gpu training may lead to performance deduction.
> 
> You can find more details for reproduction in this issue: #6

thank you, we will try again

**Comment by Saralyliu** [comment]
> Please use 1 GPU to train our model. The multi-gpu training may lead to performance deduction.
> 
> You can find more details for reproduction in this issue: #6

Hi, we follow the setting of  issue:#6, and only get 60.1 mIoU on voc group 0. And we make a comparison with your train and test log, we cannot get a similar training log, we find that the closer result 61.4 Is get at 43 epoch and 61.14 at 119 epoch, and the best result is not the final epoch. If possible, could you upload your completed training parameters . Thank you very much.
